baracoda/operations.py

```diff
@@ -167,3 +173,41 @@ def __set_prefix_item(self):
             prefix item or None if prefix does not exist
         """
         self.prefix_item = get_prefix_item(self.prefix)
+
+
+class ChildBarcodeOperations:
```

Comment:
This class only has one method and it is static, which is equivalent to having a function inside a module.